baremetal/metal3machine_manager.go

```diff
@@ -533,8 +533,29 @@ func (m *MachineManager) Delete(ctx context.Context) error {
 			host.Spec.NetworkData = nil
 			bmhUpdated = true
 		}
-		if host.Spec.Online {
+
+		//	turn on/off bmh depending on AutomatedCleaningMode and fastTrack values
+
+		//	Automated Cleaning | Fast Track	| BMH
+		//	disabled			false 		turn off
+		//	disabled			true 		turn off
+		//	metadata			false 		turn off
+		//	metadata 			true 		turn on
```

Comment:
It is a bit difficult to see which values go with which column, fix the alignment.

Reply:
done